src/components/connect.js

```diff
@@ -1,5 +1,5 @@
-import 'whatwg-fetch'
```

Comment:
can we move `'whatwg-fetch'` from `peerDependencies` into `devDependencies` and make a note in the readme (and an invariant in code) to make it the app's responsibility to provide fetch? This should also deliver #11 since app's can just use `isomorphic-fetch` instead.